nimsuggest/nimsuggest.nim

```diff
@@ -775,6 +775,9 @@ else:
     myLog(conf, "START " & conf.projectFull.string)
 
     var graph = newModuleGraph(cache, conf)
+    graph.onMarkUsed = proc (g: ModuleGraph; info: TLineInfo; s: PSym; usageSym: var PSym; isDecl: bool) =
+      suggestSym(g, info, s, usageSym, isDecl)
+    graph.onSymImport = graph.onMarkUsed # same callback
```

Comment:
What's the purpose of this change?

Reply:
the library usage lose these callbacks since https://github.com/nim-works/nimskull/pull/363

Reply:
Ah, I see, thanks. However, it's not something related to the import suggestion changes, no?

Reply:
yes.